cdap-examples/SparkKMeans/src/main/scala/co/cask/cdap/examples/sparkkmeans/SparkKMeansProgram.scala

```diff
@@ -31,24 +31,8 @@ import org.slf4j.{Logger, LoggerFactory}
  * Implementation of KMeans Clustering Spark Program.
  */
 class SparkKMeansProgram extends ScalaSparkProgram {
-  private final val LOG: Logger = LoggerFactory.getLogger(classOf[SparkKMeansProgram])
-
-  def pointVector(point: Point): Vector[Double] = {
-    DenseVector(Array(point.getX, point.getX, point.getZ).map(_.doubleValue()))
-  }
 
-  def closestPoint(p: Vector[Double], centers: Array[Vector[Double]]): Int = {
-    var bestIndex = 0
-    var closest = Double.PositiveInfinity
-    for (i <- 0 until centers.length) {
-      val tempDist = squaredDistance(p, centers(i))
-      if (tempDist < closest) {
-        closest = tempDist
-        bestIndex = i
-      }
-    }
-    bestIndex
-  }
+  import SparkKMeansProgram._
```

Comment:
That's the way to import companion object so that I can do:

```scala
  LOG.info(...)
```
instead of
```scala
  SparkKMeansProgram.LOG.info(...)
```